src/test/modules/delay_execution/meson.build

```diff
@@ -12,9 +10,10 @@ endif
 
 delay_execution = shared_module('delay_execution',
   delay_execution_sources,
-  kwargs: pg_mod_args,
+  kwargs: pg_mod_args_no_install,
 )
 testprep_targets += delay_execution
+test_install_libs += delay_execution
```

Comment:
I think we don't need to assign both of these variables. How about adding test_install_libs to testprep_targets as a whole, instead of repeating these lines all over?